syntax = "proto3";

service Interact {
  // Sends a greeting
  rpc HandleInteract (InteractReq) returns (InteractRes) {}
}

message InteractReq {
  string id = 1;
  int64 time = 2;
}

message InteractRes {
  int32 code = 1;
  string id = 2;
}
